ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -9,23 +9,15 @@
         </div>
         <div class="form-group" [ngClass]="{'has-error': titleIsDuplicated(projectTitleControl) || hasError(projectTitleControl)}">
             <label class="control-label" for="idProjectTitle">* Name</label>
-            <input #projectTitleControl="ngModel"
-                   name="projectTitle"
-                   id="idProjectTitle"
+            <input type="text" name="projectTitle" id="idProjectTitle" placeholder="my-project"
+                   required pattern=".*[^\s].*" minlength="1" maxlength="128"
+                   #projectTitleControl="ngModel"
```

Comment:
Because the validator is involved in the PR. I was just trying to find some sense in the mess.